datadog_cluster_agent/README.md

```diff
@@ -10,43 +10,47 @@ Follow the instructions below to install and configure this check for an Agent r
 
 ### Installation
 
-The Datadog-Cluster-Agent check is included in the [Datadog Agent][2] package.
+The Datadog Cluster Agent check is included in the [Datadog Agent][2] package.
 No additional installation is needed on your server.
 
 ### Configuration
+The Datadog Cluster Agent check will use [Autodiscovery][3] to automatically configure itself in most scenarios. This check will run in the Datadog Agent pod on the same node as the Cluster Agent pod. It will not run in the Cluster Agent itself.
```

Comment:
```suggestion
The Datadog Cluster Agent check uses [Autodiscovery][3] to automatically configure itself in most scenarios. The check runs in the Datadog Agent pod on the same node as the Cluster Agent pod. It will not run in the Cluster Agent itself.
```